index.php

```diff
@@ -23,8 +23,9 @@
 
 		if ( is_home() && ! empty( $sticky ) && ! is_paged() ) :
 			// Use the last added sticky post - last in array.
-			$the_query         = new WP_Query( array( 'post__in' => $exclude_from_grid ) );
-			$exclude_from_grid = array( $sticky[ count( $sticky ) - 1 ] );
+			// We must use ignore_sticky_posts alongside post__in, because
+			// WP_Query will always retrieve all sticky posts vs just the latest.
+			$the_query = new WP_Query( array( 'post__in' => $exclude_from_grid, 'ignore_sticky_posts' => true ) );
```

Comment:
We don't have PHPCS set up here, which we should fix (#4) but I'd still recommend spacing this out for legibility
```suggestion
			$the_query = new WP_Query(
				array(
					'post__in'            => $exclude_from_grid,
					'ignore_sticky_posts' => true
				)
			);
```

Reply:
I agree, thank you for pointing that out.